tlslite/keyexchange.py

```diff
@@ -274,6 +275,9 @@ def processClientKeyExchange(self, clientKeyExchange):
             raise TLSLocalAlert(AlertDescription.illegal_parameter,
                                 "Invalid dh_Yc value")
 
+        if dh_Yc % self.dh_p == 0:
+            raise TLSIllegalParameterException("Suspicious dh_Yc value")
```

Comment:
yes, delete that condition

no, don't add a test case, I've done it in #121 just waiting for code review